app/controllers/forms_controller.rb

```diff
@@ -0,0 +1,26 @@
+class FormsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @forms = Form.all
+  end
+
+  def new
+    @form = Form.find_by(_type: params[:_type])
+  end
+
+  def create
+    debugger
```

Comment:
remove debugger also try to commit clean code without commented code and debugger